app/src/main/java/com/github/libretube/ui/sheets/ExpandedBottomSheet.kt

```diff
@@ -36,4 +29,18 @@ open class ExpandedBottomSheet : BottomSheetDialogFragment() {
         // ensure that the sheet doesn't hide parts of the video
         dialog?.dismiss()
     }
+
+    fun expand() {
+        bottomSheet?.let { fl ->
+            val bottomSheetInfoBehavior = BottomSheetBehavior.from(fl)
+            bottomSheetInfoBehavior.state = BottomSheetBehavior.STATE_EXPANDED
+        }
+    }
+
+    fun halfExpand() {
```

Comment:
I think we should instead add a parameter called `expandHalf` (boolean) to the function `expand` instead of creating two different functions doing basically the same thing.

Reply:
You are right, I had want to hide it, but I find it will just dismiss the dialog, so I change it to halfExpand